### PR TITLE
[e2e] - Fix Bitrise pipeline environment variable

### PR DESCRIPTION
**Pull request by Andepande**
## **Description**

- Small e2e fix in Bitrise yaml file, adding METAMASK_ENVIRONMENT env variable.

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by tommasini — DISMISSED** [review]
LGTM! This will only affect detox builds

File: e2e/specs/wallet/wallet-tests.spec.js [hunk 1]
```diff
@@ -82,19 +82,19 @@ describe(Smoke('Wallet Tests'), () => {
     await NetworkEducationModal.isNotVisible();
   });
 
-  it('should add a token via token autocomplete', async () => {
```
Comment (tommasini) [inline e2e/specs/wallet/wallet-tests.spec.js:85]: Why are we commenting this test case?

**Review by tommasini — APPROVED** [review]
LGTM for we can enable the e2e pipeline

File: e2e/specs/wallet/wallet-tests.spec.js (hunk 1 above)
Comment (Andepande) [reply e2e/specs/wallet/wallet-tests.spec.js:85]: Requires further investigation as to why it's failing in a separate PR

**Pull request merged**